angular-growl-v2/angular-growl-v2.d.ts

```diff
@@ -211,4 +211,39 @@ declare module angular.growl {
          */
         position(): string;
     }
+
+    /**
+     * GrowlMessages service.
+     */
+    interface IGrowlMessagesService {
+        /**
+         * Initialize a directive
+         * We look at the preloaded directive and use this else we
+         * create a new blank object
+         * @param referenceId
+         * @param limitMessages
+         */
+        initDirective(referenceId: number, limitMessages: boolean): ng.IDirective;

```

Comment:
Are you sure this is boolean? Look here: https://github.com/JanStevens/angular-growl-2/blob/master/src/growlMessageService.js#L122